activate_and_execute.sh

```diff
@@ -24,6 +24,9 @@ if [ -f "$ENV_FILE" ]; then
     bash "$SCRIPT_DIR"/import_vars.sh "$ENV_FILE"
 fi
 
+# check wandb and login
```

Comment:
```suggestion
# Login to Weights & Biases, if the wandb python package is installed 
# and the environment variable WANDB_API_KEY is set.
```

Reply:
fixed in [d811634](https://github.com/DFKI-NLP/pegasus-bridle/pull/2/commits/d81163421cd07ad2ee652fcca4026e9425881485)